client/components/tinymce/plugins/simple-payments/dialog/product-image-picker.jsx

```diff
@@ -102,6 +102,7 @@ class ProductImagePicker extends Component {
 						onClose={ this.setImage }
 						enabledFilters={ [ 'images' ] }
 						visible={ this.state.isSelecting }
+						additionalBackdropClassNames="remove-backdrop-background"
```

Comment:
perhaps `backdropOpacity` or `isBackdropVisible`?

Reply:
If we stick with a class name, perhaps let's keep with the modifier pattern like `is-hidden` for example https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#csssass-coding-guidelines

Reply:
Updated in ed65d2d03f38efb2fc97b22a621c9d2c60678fda.